tensorflow_probability/python/positive_semidefinite_kernels/polynomial_test.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2018 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Tests for Polynomial and Linear."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+from absl.testing import parameterized
+import numpy as np
+
+
+import tensorflow as tf
+from tensorflow.python.framework import test_util
+from tensorflow_probability import positive_semidefinite_kernels as psd_kernels
```

Comment:
Move this up a line.

Reply:
I think the other linter complained that I needed to put `import tensorflow as tf` and `from tensorflow.python.framework import test_util` together.
Will update as you suggested